src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-kafka-confluentcloud-k8s.mdx

```diff
@@ -0,0 +1,281 @@
+                  `CONFLUENT_API_KEY`
+                </td>
+                <td>
+                  Cloud API key
+                </td>
+                <td>
+                  [Cloud API key docs](https://docs.confluent.io/cloud/current/monitoring/metrics-api.html#metrics-quick-start)
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  `CONFLUENT_API_SECRET`
+                </td>
+                <td>
+                  Cloud API secret
+                </td>
+                <td>
+                  [Cloud API key docs](https://docs.confluent.io/cloud/current/monitoring/metrics-api.html#metrics-quick-start)
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  `CONNECTOR_ID`
+                </td>
+                <td>
+                  (OPTIONAL) You can monitor your Confluent connectors by specifying the ID here
+                </td>
+                <td>
+                  [Docs for list connector ID command](https://docs.confluent.io/confluent-cli/current/command-reference/connect/cluster/confluent_connect_cluster_list.html)
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  `SCHEMA_REGISTRY_ID`
+                </td>
+                <td>
+                  (OPTIONAL) You can monitor your Confluent Schema Registry by specifying the ID here
+                </td>
+                <td>
+                  [Docs for list connector ID command](https://docs.confluent.io/confluent-cli/current/command-reference/schema-registry/schema/confluent_schema-registry_schema_list.html)
+                </td>
+              </tr>
+            </tbody>
+        </table>
+
+    </Step>
+    <Step>
+
+       ## View your data in New Relic
+
+      You can view your Confluent Cloud data in a few different ways.
+
+      * Navigate to the [New Relic marketplace](https://one.newrelic.com/marketplace) and search for `Confluent`. The available dashboards can be installed right onto your account!
+      * Navigate to the metrics explorer and filter for `confluent_kafka`. This data can be added to any custom alert or dashboard.
+
+
+    </Step>
+</Steps>
+
+### Confluent Cloud metrics [#confluent-metrics]
+
+This integration covers all the *Exportable* metrics within the [Confluent Cloud Metrics API](https://api.telemetry.confluent.cloud/docs/descriptors/datasets/cloud). We have a partial list of the *Exportable* metrics below:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        Name
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        confluent_kafka_server_received_bytes
+      </td>
+      <td>
+        The delta count of bytes of the customer's data received from the network. Each sample is the number of bytes received since the previous data sample. The count is sampled every 60 seconds.
```

Comment:
```suggestion
        The delta count of bytes of your data received from the network. Each sample is the number of bytes received since the previous data sample. The count is sampled every 60 seconds.
```